### PR TITLE
Add humidity endpoint

### PR DESCRIPTION
**Pull request by Paranoid93**
This PR adds a humidity endpoint to the project, but it also renamed the module and files - this needs to be discussed (and probably reverted)

### CODE REVIEW

**Comment by buxxi** [comment]
Cool, will take a deeper look into this weekend.

Name might be fine, a little bit long maybe :)

But maybe temp and humidity should be the same endpoint and not 2 separate?

**Comment by Paranoid93** [comment]
In my usecase I'm sending those values independently (and only on update), which is why I added a second endpoint. 

Yeah the name is long :D but I thought just adding "Humiditiy" everywhere would describe the whole featureset then already. Depending on the name, the package.json also needs to be changed

**Comment by buxxi** [comment]
So, been thinking a bit about it.

Since the name change basically will force the user (probably only you and me) to reinstall it there's no need to keep it backwards compatible.

My suggestion for the name is `MMM-IndoorClimate-Endpoint` (future proofing if there ever will be like air quality or something later). So the endpoint should be named `indoor-climate` then.

I would like to have it as one endpoint with both `temp` and `humidity` being optional so it's only one call needed if both are available.

1. If both `temp` and `humidity` is null in the payload, then return 400.
2. If both are set, trigger both `INDOOR_TEMPERATURE` and `INDOOR_HUMIDITY`
3. If one is not set in the payload, don't trigger that event

If you make those changes I can merge it and rename the repository and update it in the 3rd party module list.

**Comment by Paranoid93** [comment]
still gotta test it and reconfigure everything in my setup

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-    "name": "mmm-indoortemperature-endpoint",
+    "name": "mmm-indoorclimate-endpoint",
     "version": "1.0.0",
-    "description": "A module for MagicMirror that can receive a POST-request for indoor temperature and forwards it as an event internally",
+    "description": "A module for MagicMirror that can receive a POST-request for indoor climate values like temperature or humidty and forwards it as an event internally",
     "license": "MIT",
     "dependencies": {},
     "repository": {
```
Comment (Paranoid93) [inline package.json:7]: the repository url should still be updated, if you shift this into a new repo

File: package.json
Comment (Paranoid93) [inline package.json:1]: those values will still need an update if you shift this into a new repo/rename it
 ```
    "repository": {
        "type": "git",
        "url": "https://github.com/buxxi/MMM-IndoorTemperature-Endpoint.git"
    },
"homepage": "https://github.com/buxxi/MMM-IndoorTemperature-Endpoint"
```

**Comment by buxxi** [comment]
Thank you for the contribution

**Pull request merged**